src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -355,6 +363,91 @@ def acr_task_update(cmd,  # pylint: disable=too-many-locals
     return client.update(resource_group_name, registry_name, task_name, taskUpdateParameters)
 
 
+def acr_task_identity_assign(cmd,
+                             client,
+                             task_name,
+                             registry_name,
+                             assign_identity,
+                             resource_group_name=None):
+    _, resource_group_name = validate_managed_registry(
+        cmd, registry_name, resource_group_name, TASK_NOT_SUPPORTED)
+        
+    identity = None
+    if assign_identity is not None:
+        identity = _build_identities_info(cmd, assign_identity)
+
+    TaskUpdateParameters, PlatformUpdateParameters, AgentProperties, TriggerUpdateParameters = cmd.get_models(
+        'TaskUpdateParameters', 'PlatformUpdateParameters', 'AgentProperties', 'TriggerUpdateParameters')
+    taskUpdateParameters = TaskUpdateParameters(
+        identity=identity,
+        status=None,
```

Comment:
Are these `None` values placeholders? If not, you don't need to assign `None` since they default to `None`.